docs/UserGuide.md

```diff
@@ -393,6 +436,11 @@ If unindicated, the `School Residence` field will default to `DOES NOT LIVE ON C
 <br>
 **A**: Allergies and past medical history.
 
+**Q**: Why is matriculation number used to identify a student? It is difficult to remember! <br>
+**A**: Every student has a unique matriculation number. This guarantees that there will not be duplicate students in the records, which would make tracking records more tedious.  
+
+### Managing appointments
+
```

Comment:
Maybe can remove the 'It is difficult to remember!' or change the tone of it. I don't think its a good idea to come across as so aggressive.

Reply:
I think can remove since you dont put it in the appointments question as well?

Reply:
Wait so you want me to remove what again?